string scene_name
string map_name
---
uint32 error_code
